File: Introduction to VB/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;


namespace Introduction_to_VB
{
    public partial class Form1 : Form
    {
        
        private Form2 form2Instance;

        public Form1()
        {
            InitializeComponent();
        }

        private void loadobjectform_Click(object sender, EventArgs e)
        {
            
            form2Instance = new Form2();
            form2Instance.Show();
        }

        private void button2_Click(object sender, EventArgs e)
        {
            
            this.Close();
        }
    }
}

File: Introduction to VB/Form1.vb
Imports System
Imports System.Windows.Forms


Namespace Introduction_to_VB
    Public Partial Class Form1
        Inherits Form

        Private form2Instance As Form2

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub loadobjectform_Click(sender As Object, e As EventArgs)

            form2Instance = New Form2()
            form2Instance.Show()
        End Sub

        Private Sub button2_Click(sender As Object, e As EventArgs)

            Close()
        End Sub
    End Class
End Namespace
